### PR TITLE
Implementing Video-page UI

### PR DESCRIPTION
**Pull request by sanjayk0508**
## Video page preview

https://github.com/notaryproject/notaryproject.dev/assets/102804548/67ccbcfb-4eae-4298-87ad-792101d4a889

### This PR contains

- **Notary Project YouTube Videos:** The majority of our videos consist of informative presentations.
- **Developer-Friendly UI/UX:** The user interface and experience are designed to facilitate seamless navigation for developers, ensuring easy access to relevant video content (with a Search bar). It has a Mobile-responsive UI.
- **Detailed Information:** Each video comes with a summary, view count, duration, and upload date. And along with a button that allows users to open the video in a new tab, directly accessing the YouTube link.
- **Editing and Adding new videos:** `content/videos` folder which contains `.md` files can be useful for adding more new videos.

### Preview link
[https://deploy-preview-278--notarydev.netlify.app/videos/](https://deploy-preview-278--notarydev.netlify.app/videos/)

### CODE REVIEW

**Comment by PranitPatil03** [comment]
Hello @sanjayk0508 

The video page looks good, but it could be improved. Consider adding several containers with multiple videos on different screens. The first screen could have 5-6 videos, and subsequent screens could follow a similar pattern. This would address the issue of the UI overlapping with the footer section.

<details><summary>Details</summary>
<p>

image

</p>
</details> 

<details><summary>Use this as a reference</summary>
<p>

image


</p>
</details>

**Comment by sanjayk0508** [comment]
@PranitPatil03 Thank you for addressing the issue. The main component's fixed height resulted in overriding the footer.

I have considered implementing multiple sections for the video page to fit in additional videos. For a better user experience, we can incorporate a category component that filters out specific types of videos.

File: config.yaml [hunk 1]
```diff
@@ -184,3 +184,6 @@ menu:
     - name: Roadmap
       url: https://github.com/notaryproject/roadmap
       weight: -20
+    - name: Video
```
[the remark this answers is not in the document]
Comment (sanjayk0508) [reply config.yaml:187]: Done!

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@iamsamirzon @toddysm @yizha1 Would you mind reviewing this PR? I think it's good to merge it as the first version of the video page

**Comment by iamsamirzon** [comment]
IMO, we need to put the videos in a chronological order, so consumers do not have to guess what is more recent. should not There is lot of content, including Notary with TUF and Notation, and Notary v2 related video, so we need a  para up there explaining that are the main categories, and let consumers filter what they are interested in ( Mainly Notation).

**Review by toddysm — CHANGES_REQUESTED** [review]
+1 to what @iamsamirzon feedback is. Also, the menu item is called "Video" while the page is called "Videos" - this is confusing

File: content/en/videos/video11.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Notary v2 Outstanding Issues Working Session - Justin Cormack"
```
Comment (FeynmanZhou) [inline content/en/videos/video11.md:2]: Let's remove the speaker's name from the title as the video cover includes the speaker's name. If we keep the speaker name, please make sure all speaker's names are there.

File: content/en/videos/video12.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Notary v2 Introduction and Status Report - Justin Cormack, Docker"
```
Comment (FeynmanZhou) [inline content/en/videos/video12.md:2]: Same as above

File: content/en/videos/video5.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Intro: TUF/Notary - Justin Cappos, NYU & Justin Cormack, Docker"
```
Comment (FeynmanZhou) [inline content/en/videos/video5.md:2]: Same as above

File: content/en/videos/video7.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Deep Dive: TUF / Notary - Justin Cappos & Lukas Puehringer, NYU"
```
Comment (FeynmanZhou) [inline content/en/videos/video7.md:2]: Same as above

File: content/en/videos/video8.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Notary v2: Supply Chain Security for Containers - Justin Cormack, Docker"
```
Comment (FeynmanZhou) [inline content/en/videos/video8.md:2]: Same as above

**Review by FeynmanZhou — COMMENTED** [review]
@sanjayk0508 PTAL

File: content/en/videos/video11.md (hunk 2 above)
Comment (sanjayk0508) [reply content/en/videos/video11.md:2]: Since the video thumbnail includes the speaker's name, there is no need to mention it again. I will look into it.

**Comment by yizha1** [comment]
@sanjayk0508 Would you be able to resolve the comments?

**Comment by sanjayk0508** [comment]
@yizha1 Sure. We need to implement the redesign of our Home page first. The reason is that the menu bar has many sub-pages title, so we should create a separate PR to update the menu bar after implementing the Home page redesign.

File: content/en/videos/video15.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Improve Vulnerability Management with OCI Artifacts -It Is That Easy!"
```
Comment (FeynmanZhou) [inline content/en/videos/video15.md:2]: To simplify this PR review process, I would suggest keeping this video only. Other videos can be updated by other maintainers.

**Pull request closed without merging**